frontend/src/components/listofusers/DeleteModalButton.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react'
+import { Button, Modal } from 'react-bootstrap'
+import { useTranslation } from 'react-i18next/'
+import DeletionFailedModal from './DeletionFailedModal'
+import { deleteUserRegistration } from '../../communications/userApi'
+import useUsers from '../useUsers'
+
+const DeleteModalButton = ({ userId }) => {
+  const [showConfirmDeletion, setShowConfirmDeletion] = useState(false)
+  const [showFailureModal, setShowFailureModal] = useState(false)
+  const { fetchUsers } = useUsers()
+  const { t } = useTranslation()
+
+  const deleteUser = async () => {
+    setShowConfirmDeletion(false)
+    const responseStatus = await deleteUserRegistration(userId)
+
+    if (responseStatus === 200) {
```

Comment:
Nem szép itt kezelni HTTP kódot, ezt rejtse el az API réteg. Ha nem 200, akkor dobjon exceptiont, amit itt kapjunk el. Amúgy sincs itt megfelelően kezelve, ha elhal a kérés, például megszakad a net, akkor is exception keletkezik.

Reply:
Átszerveztem. Valóban, a lehetőségek szűk keresztmetszetét kezeltem csak le, amit egy !== 200 jórészt megoldott volna.